integration-tests/config.ts

```diff
@@ -140,6 +141,15 @@ const oxfordnetEphemeral: Config =
 const oxfordnetSecretKey: Config =
   { ...oxfordnetEphemeral, ...{ signerConfig: defaultSecretKey }, ...{ defaultRpc: 'http://ecad-oxfordnet-full:8732' } };
 
+const nairobinetSecretKey: Config = 
```

Comment:
perhaps the naming of the constant can be something like nairobiSandbox / nairobiFlextesa? think it'll be helpful to distinguish from testnet

Reply:
the 'secretKey' part implies that its running on a sandbox I think Daniele or Roxane created the term, I don't have any problems with the current naming scheme.